client/src/resources/cert-and-project-map.ts

```diff
@@ -459,38 +459,38 @@ const certMap = [
   {
     id: '606243f50267e718b1e755f4',
     title: 'Relational Databases',
-    certSlug: SuperBlocks.RelationalDb,
-    flag: 'isRelationalDatabasesCert',
+    certSlug: 'relational-databases-v8',
```

Comment:
```suggestion
    certSlug: SuperBlocks.RelationalDb
```
Shouldn't that be the case for all these certSlugs?  It breaks the pattern, otherwise.

Reply:
Pretty sure there's a reason I didn't go that route. Let me get back to you on it.

Reply:
Yea, the certSlug string here doesn't align with the [SuperBlocks string here](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/config/certification-settings.ts#L20). The rest of the certs in cert-and-project-map that have v7/v8 at the end don't link to that SuperBlocks object either. We could probably add another object there for the slugs since they differ from the name of the SuperBlock now. Not on this PR though.